docs/UserGuide.md

```diff
@@ -269,6 +252,43 @@ Example:
   <br>
   </p>
 
+<div style="page-break-after: always;"></div>
+
+#### Deleting a contact : `delete`
+
+*Maybe if you want to tidy up your contacts.*
+
+Deletes the specified contact from SOCket.
+
+Format: `delete INDEX`
+
+* Deletes the contact at the specified `INDEX`.
+* The index refers to the index number shown in the displayed contact list.
+
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
+You can use the find command or list command to narrow down your contact list first before performing delete based on the filtered list.
+</div>
+
+Examples:
+* `list` followed by `delete 2` deletes the 2nd contact in SOCket.
+* `find Betsy` followed by `delete 1` deletes the 1st contact in the results of the `find` command.
+
+#### Clearing all contacts or tags : `clear`
+
+*Deleting contacts one by one is a hassle, clear can solve that hassle.*
+
+Clears all contacts from SOCket based on the given tag(s); if tags are not included, clears all contacts in SOCket.
+
+Format: `clear [t/TAG]...`
+
+* Removes all contacts with the specified tags.
+* Any tag included is **case-insensitive**.
+  * e.g. `t/CS2103T` is equivalent to `t/cs2103t`.
+* The tags **must exist** in SOCket.
+* If tags are provided, only remove existing tags.
+  * e.g. `clear t/cs2103t t/cs2103` will only remove the contacts associated with the tag `cs2103t` if there exists the `cs2103t` tag but not `cs2103` in SOCket.
+* If no tag field is provided, remove all contacts in SOCket.
+* A confirmation prompt will appear before removal of contacts.
```

Comment:
Was under the impression that having CRUD commands together was logical, could you perhaps elaborate on this formatting decision?

Reply:
I mainly shifted up find and list as I have added a tip in delete which references both find and list.

My main thought was that we should introduce the command first before including them in the tip rather than mentioning a command in a tip which a reader would not have read yet if he/she would to read from top to bottom.